docs/integrations/databases/fauna.mdx

```diff
@@ -3,112 +3,165 @@ title: Integrate Fauna with Clerk
-
-```jsx
-import React from 'react';
-import { useAuth } from '@clerk/nextjs';
-import { Client, fql } from "fauna";
-
-const Example = () => {
-  const { getToken } = useAuth();
-  const [message, setMessage] = React.useState('');
-
-  const makeQuery = async () => {
-    let client;
-    try {
-      const secret = await getToken({ template: '<your-template-name>' });
-      client = new Client({ secret: secret });
-      const response = await client.query(fql`'Hello World!'`);
-      setMessage(response);
-    } catch (error) {
-      console.error(error);
-      setMessage('Error occurred');
-    } finally {
-      if (client) client.close();
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    },
+    {
+      title: "Integrate the appropriate Clerk SDK in your local project",
+      link: "/docs/quickstarts/overview",
+      icon: "code-bracket",
+    },
+  ]}
+>
+  - Configure Fauna to accept JWTs from Clerk
+  - Create a JWT template in Clerk to generate Fauna JWTs
+  - Use the Fauna JWTs to authenticate Fauna queries in your Next.js app
+</TutorialHero>
+
+> [!IMPORTANT]
+> Check out the [demo repo](https://github.com/clerk/clerk-fauna-nextjs) for a full example of how to integrate Fauna with Clerk in a Next.js app.
+
+Integrating Fauna with Clerk gives you the benefits of using a Fauna database while leveraging Clerk's authentication, prebuilt components, and webhooks.
+
+This guide will walk you through the steps to integrate Fauna with Clerk in your Next.js app.
+
+<Steps>
+  ### Get your Clerk Frontend API URL and JWKS URL
+
+  <SignedIn>
+    Add the following keys to your `.env.local` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
+  </SignedIn>
+
+  <SignedOut>
+    1. Navigate to the Clerk Dashboard.
+    1. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
+    1. In the left sidebar, select **Show API URLs**.
+    1. Copy the **Frontend API URL**.
+    1. In the left sidebar, select **Show JWT public key**.
+    1. Copy the **JWKS URL**.
+    1. Paste your keys into your `.env.local` file.
+
+    The final result should resemble the following:
+  </SignedOut>
+
+  ```env {{ filename: '.env.local' }}
+  NEXT_PUBLIC_CLERK_FRONTEND_API_URL={{fapi_url}}
+  NEXT_PUBLIC_CLERK_JWKS_URL={{jwks_url}}
+  ```
+
+  ### Configure Fauna
+
+  1. Navigate to the [Fauna Dashboard](https://dashboard.fauna.com/) and select your database.
+  1. Select the **Access Providers** tab and select **Create Access Provider**.
+  1. Fill out the form:
+     - Enter a **Name** for the access provider, such as Clerk.
+     - Copy and save the **Audience** URL. You will need this later.
```

Comment:
We bold titles/proper names. In the Fauna Dashboard, it is called the "Audience" value, not "Audience URL". So here in the copy, we could written "save the **Audience** value** or "save the **Audience** text" or really anything, but its a URL so I wrote "save the **Audience** URL"
But "URL" isn't actually part of the name, so it doesn't get bolded.
Whereas, when URL is bolded in other places in this doc, it is because "URL" is part of the actual name, such as "Frontend API URL"